src/panels/lovelace/cards/hui-alarm-panel-card.ts

```diff
@@ -191,7 +191,7 @@ class HuiAlarmPanelCard extends LitElement implements LovelaceCard {
           : html`
               <paper-input
                 id="alarmCode"
-                label="Alarm Code"
+                .label="${this.hass.localize('ui.card.alarm_control_panel.alarm_code')}"
```

Comment:
I don't see `alarm_code` in the translations?

Reply:
@bramkragten Could you please point me where should I add it?

Reply:
`/src/translations/en.json`

Reply:
Done.